src/time_zone_lookup_test.cc

```diff
@@ -12,8 +12,6 @@
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
 
-#include "cctz/time_zone.h"
```

Comment:
This was here intentionally, so that we can test that `cctz/time_zone.h` includes what it needs.  That is, it is the "main include" (category 0).  I guess we'll have to do something to teach clang-format that, but it could wait.